cellprofiler/modules/resizeobjects.py

```diff
@@ -185,12 +222,17 @@ def upgrade_settings(self, setting_values, variable_revision_number, module_name
         if variable_revision_number == 1:
             setting_values += ["None"]
             variable_revision_number = 2
+
+        if variable_revision_number == 2:
+            setting_values = (
+                setting_values[:1] + ["0.25", "0.25", "0.25"] + setting_values[3:4] + ["10"] + setting_values[5:]
```

Comment:
`setting_values[:1] + ["0.25", "0.25", "0.25"] + setting_values[3:4] + ["10"] + setting_values[5:]`

I think here your second one you want `2:4`, and then your third one you just want `4` or `-1`

Reply:
(As above, check me!)

Reply:
You are right, I will make the changes